### PR TITLE
docs(identity): add new environment variable for components' client id

### PR DESCRIPTION
**Pull request by maryarm**
## When should this change go live?

- [x] This change is not yet live and should not be merged until January 8.4 release (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — DISMISSED** [review]

File: docs/self-managed/identity/deployment/configuration-variables.md [hunk 1]
```diff
@@ -33,16 +33,19 @@ methods.
...
 
-| Environment variable                 | Description                                    | Default value |
-| ------------------------------------ | ---------------------------------------------- | ------------- |
-| `KEYCLOAK_INIT_<COMPONENT>_SECRET`   | The secret used for authentication flows.      | No default    |
-| `KEYCLOAK_INIT_<COMPONENT>_ROOT_URL` | The root URL of where the component is hosted. | No default    |
+| Environment variable                  | Description                                     | Default value |
+| ------------------------------------- | ----------------------------------------------- | ------------- |
+| `KEYCLOAK_INIT_<COMPONENT>_SECRET`    | The secret used for authentication flows.       | No default    |
+| `KEYCLOAK_INIT_<COMPONENT>_ROOT_URL`  | The root URL of where the component is hosted.  | No default    |
+| `KEYCLOAK_INIT_<COMPONENT>_CLIENT_ID` | The client to create and use for the component. | `<component>` |
```
Comment (Ben-Sheppard) [inline docs/self-managed/identity/deployment/configuration-variables.md:40]:
```suggestion
| `KEYCLOAK_INIT_<COMPONENT>_CLIENT_ID` | The client to create and use for the component. | `<COMPONENT>` |
```

File: docs/self-managed/identity/deployment/configuration-variables.md [hunk 2]
```diff
@@ -33,16 +33,19 @@ methods.
...
 
 :::note
 Identity supports the following values for the `<COMPONENT>` placeholder: `OPERATE`, `OPTIMIZE`, `TASKLIST`,
 and `WEBMODELER`.
 
 For the `WEBMODELER` value, only the `KEYCLOAK_INIT_<COMPONENT>_ROOT_URL` variable is required to be set.
+
+For the `KEYCLOAK_INIT_<COMPONENT>_CLIENT_ID`, the default value is the component name in lowercase except for `WEBMODELER`, set to `web-modeler`.
```
Comment (Ben-Sheppard) [inline docs/self-managed/identity/deployment/configuration-variables.md:48]:
```suggestion
For `KEYCLOAK_INIT_<COMPONENT>_CLIENT_ID`, the default value is the component name in lowercase except for `WEBMODELER`, set to `web-modeler`.
```
Or 
```suggestion
For the `KEYCLOAK_INIT_<COMPONENT>_CLIENT_ID` value, the default is the component name in lowercase except for `WEBMODELER`, which is`web-modeler`.
```

**Review by Ben-Sheppard — APPROVED** [review]
Two small NABD comments for consistency but looks good to me :)

**Comment by christinaausley** [comment]
@maryarm Would you like this to go out with release next week? If so, just tidy up the remaining comments and I'll get this approved and merged 👍

**Comment by maryarm** [comment]
> @maryarm Would you like this to go out with release next week? If so, just tidy up the remaining comments and I'll get this approved and merged 👍

@christinaausley I've applied the comments and it's ready for your review. Thanks 👍

**Review by Ben-Sheppard — APPROVED** [review]
Content wise this looks good to me :)

**Pull request merged**